diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
index 757f3ac..08e787a 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
@@ -18,7 +18,7 @@ import Animated, {
   useAnimatedStyle,
   useDerivedValue,
   runOnJS,
-  interpolate,
+  newInterpolate as interpolate,
   Extrapolate,
   runOnUI,
   useWorkletCallback,
